### PR TITLE
[16.0][OU-ADD] pad_project: Nothing to do

### PR DESCRIPTION
**Pull request by daiduongnguyen-odoo**
Obsolete module, use directly in odoo itself

Here is the PR that merged the feature `https://github.com/odoo/odoo/pull/75768` and here the PR that delete it `https://github.com/odoo/odoo/pull/76467`

### CODE REVIEW

File: docsource/modules150-160.rst [hunk 1]
```diff
@@ -534,7 +534,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | |del| pad                                       |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| pad_project                               |                      |                                                 |
+| |del| pad_project                               |Nothing to do         |Obsolete module, use directly in odoo itself     |
```
Comment (legalsylvain) [inline docsource/modules150-160.rst]:
```suggestion
| |del| pad_project                               | Done         |Obsolete module, use directly in odoo itself     |
```


Nothing to do should be used when when there is nothing to do ;-)

**Review by legalsylvain — CHANGES_REQUESTED** [review]

File: docsource/modules150-160.rst (hunk 1 above)
Comment (daiduongnguyen-odoo) [reply docsource/modules150-160.rst]: ok i see, done, and also fix the related `pad` module too

File: docsource/modules150-160.rst [hunk 2]
```diff
@@ -534,7 +534,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | |del| pad                                       |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |del| pad_project                               |                      |                                                 |
+| |del| pad_project                               |Done                  |Obsolete module, use directly in odoo itself     |
```
Comment (MiquelRForgeFlow) [inline docsource/modules150-160.rst]:
```suggestion
| |del| pad_project                               |Done                  | Merged into web_editor |
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by pedrobaeza** [comment]
This should be merged into `project`, not `web_editor`.

**Comment by daiduongnguyen-odoo** [comment]
> This should be merged into `project`, not `web_editor`.

Oki, my mistake, Done

**Pull request merged**